capstone/src/main/webapp/about.html

```diff
@@ -0,0 +1,117 @@
+    <link href="cover.css" rel="stylesheet">
+  </head>
+  <body class="text-center">
+    <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
+      <header class="masthead mb-auto">
+        <div class="inner">
+          <div class="text-center">
+            <img src="assets/perspective_horizontal_color.png" class="rounded">
+          </div>
+          <h3 class="masthead-brand">Perspective Hub</h3>
+          <nav class="nav nav-masthead justify-content-center">
+            <a class="nav-link" href="index.html">Home</a>
+            <a class="nav-link" href="text_analyzer.html">Text Analyzer</a>
+            <a class="nav-link" href="image_analyzer.html">Image Analyzer</a>
+            <a class="nav-link" href="document_analyzer.html">Document Analyzer</a>
+            <a class="nav-link" href="https://support.perspectiveapi.com/s/" target="_blank">Support</a>
+            <a class="nav-link" href="https://medium.com/the-false-positive" target="_blank">Blog</a>
+            <a class="nav-link active" href="about.html">About</a>
+          </nav>
+        </div>
+      </header>
+      <main role="main">
+        <section class="jumbotron text-center">
+          <div class="container">
+            <h1>The Creators Of Perspective Hub</h1>
+            <p class="lead text-muted">A motivated team of Google STEP interns who took on the task of making Perspective API accessible to all</p>
+            <p>
+              <a href="https://medium.com/the-false-positive" class="btn btn-primary my-2" target="_blank">Perspective Blog</a>
+            </p>
+          </div>
+        </section>
+        <div class="album py-5 bg-transparent">
+          <div class="container">
+            <div class="row">
+              <div class="col-md-4">
+                <div class="card mb-4 shadow-sm">
+                  <img class="rounded mx-auto d-block" src="assets/olu.png" alt="Olu Alamutu" style="max-width:100px;max-height:225px;"></img>
+                  <div class="card-body">
+                    <p class="card-text">Oluwadamilare Alamutu</p>
+                    <div class="d-flex justify-content-between align-items-center">
+                      <div class="btn-group">
+                        <a href="https://www.linkedin.com/in/olua/">
+                          <button type="button" class="btn btn-sm btn-outline-secondary">LinkedIn</button>
+                        </a>
+                      </div>
+                      <small class="text-muted">Binghamton '23</small>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="col-md-4">
+                <div class="card mb-4 shadow-sm">
+                  <br><img class="rounded mx-auto d-block" src="assets/chris.jpg" alt="Chris Mohri" style="max-width:100px;max-height:225px;"></img><br>
+                  <div class="card-body">
+                  <p class="card-text">Christopher Mohri</p>
```

Comment:
nit: Let's indent this line and the necessary ones below

Reply:
addressed in "indentation"